content/en/tracing/setup_overview/setup/php.md

```diff
@@ -431,7 +431,7 @@ To remove the PHP tracer:
 
 ## Troubleshooting an application crash
 
-In the unusual event of an application crash caused by the PHP tracer, typically because of a segmentation fault, the best thing to do is obtain a core dump and contact Datadog support.
+In the unusual event of an application crash caused by the PHP tracer, typically because of a segmentation fault, the best thing to do is obtain a valgrind trace or core dump and contact Datadog support.
```

Comment:
```suggestion
In the unusual event of an application crash caused by the PHP tracer, typically because of a segmentation fault, the best thing to do is obtain a core dump or a Valgrind trace and contact Datadog support.
```
Reordering to match the information below. If you want Valgrind listed first here, switch the sections around too.